PeerioTechnologies/peerio-desktop
# fix(files): don't show remove button when sharing one file

#### Relevant info and issue/PR links 
 
https://app.clubhouse.io/peerio/story/1502/when-sharing-a-file-there-s-a-button-that-does-nothing

#### Testing instructions  

Don't think it needs testing, but:

Go to files, share a file, hover over the file name and verify there's no "remove" button (screenshot is in clubhouse).

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
